src/Security/Auth0Service.php

```diff
@@ -58,7 +56,7 @@ public function __construct($api_secret, $domain, $api_identifier, $authorized_i
      *
      * @return array info as described in https://auth0.com/docs/users/concepts/overview-user-profile
      */
-    public function getUserProfileByA0UID($jwt, $a0UID)
```

Comment:
wouldn't those calling this function with 2 args now receive an error?

Reply:
No, PHP allows to pass more arguments to a function. They are just ignored (and can be accessed using `func_get_args()`).